sthttpd-cve-2017-10671/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:20.10 AS base
+
+RUN apt-get update && \
+    apt-get install -y libssl-dev
+
+FROM base AS builder
+
+RUN export DEBIAN_FRONTEND="noninteractive" && \
+    apt-get update -qq && \
+    apt-get install -y build-essential autotools-dev automake autoconf libtool git
+
+WORKDIR /sthttpd
+RUN git clone https://github.com/blueness/sthttpd . && \
+    git checkout c0dc63a49d8605649f1d8e4a96c9b468b0bff660 && \
+    git checkout HEAD^
+
+RUN ./autogen.sh && ./configure && make
+
+FROM base
```

Comment:
Does libssl-dev include the glibc-dbg symbols so that advanced_triage works?

Reply:
Not sure. I will try running with it on to find out. Is there usually a way of apt installing a version with the symbols or is a build from source kind-of-deal?